### ISSUE TITLE
No help for `circleci local execute`

### ISSUE THREAD

**Issue by rgov** [CircleCI-Public/circleci-cli#173]
On version 0.1.3469 (the latest as of this posting):

```
$ circleci local execute --help
Usage of configFlags:
  -c, --config string   config file (default ".circleci/config.yml")
Error: Unable to parse flags
```

**Comment by rgov** [comment]
Works when I rebuilt it, maybe the next release will fix this.

**Comment by bradylill** [comment]
Thanks, I've made a note of this. In the mean time you can use `circleci help local execute`

**Comment by zzak** [comment]
Hello @rgov!

This should be fixed as of [v0.1.3831](https://github.com/CircleCI-Public/circleci-cli/releases/tag/v0.1.3831), please try `circleci update` and let us know if you still have any trouble.
